### ISSUE TITLE
[Bug]: Accessibility: a few more incorrectly labeled elements when using talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [meshtastic/Meshtastic-Android#1632]
### What happened?

When using the TalkBack screen reader on Android, the following buttons are not labeled properly.

![Image](https://github.com/user-attachments/assets/6f3fd0aa-44c4-4213-aa95-9d6b6512dbf1)
![Image](https://github.com/user-attachments/assets/e65d2aec-328e-4d11-acb1-9fae227647d1)

This checkbox says nothing at all.

![Image](https://github.com/user-attachments/assets/aec21a2b-8e6b-4e54-8aee-6e7b2015fa1d)

And this button reads share when it really should say copy link.

![Image](https://github.com/user-attachments/assets/fc3db0c7-7e49-478c-be12-dcdc88e08509)

### App Version

2.5.17

### Phone

OnePlus Nord N200

### Device

Na

### Firmware

Na

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by todd2982** [comment]
I'd like to work on this one.

**Comment by shortwavesurfer2009** [comment]
I see no reason you can't. People submit PRs often and get them accepted

**Comment by todd2982** [comment]
- [X] Node sort
- [ ] Map current position
- [ ] Channel selected to share
- [ ] Copy channel URL

**Comment by shortwavesurfer2009** [comment]
Keep in mind the location button has two separate states. But otherwise, yes. Those labels sound good.

**Comment by todd2982** [comment]
The check boxes for the channels in the QR code screen read back checked/unchecked status in the emulator.  The other items have been corrected.  I'll submit the PR shortly.

**Comment by jamesarich** [comment]
fixed by #1705
